docs/scenarios/solution-authoring/vs-golden-path.md

```diff
@@ -0,0 +1,105 @@
+Building simple C# solutions with Visual Studio
+===============================================
+
+by [Bertrand Le Roy](https://github.com/bleroy) and [Phillip Carter](https://github.com/cartermp)
+
+The scripts in this document describe the steps necessary to build a number of typical .NET Core solutions, or solutions that include .NET Core components, using Visual Studio. The scenarios include testing, and using third-party libraries that have not been explicitly built for the most recent version of .NET Core.
+
+Prerequisites
+-------------
+
+These scripts assume that [Visual Studio 2015 Update 2](https://www.visualstudio.com/en-us/news/vs2015-update2-vs.aspx), and the [.NET Core RC2 SDK preview with Visual Studio tooling](https://www.microsoft.com/net/core) are installed on the machine.
+
+A solution using only .NET Core projects
+----------------------------------------
+
+### Writing the library
+
+1. File -> New -> Project. Under Templates / Visual C# / .NET Core, choose "Class Library (.NET Core)" and name the project "Library" and the solution "Golden". Leave "Create directory for solution" checked.
```

Comment:
nit: should you tell folks to open VS first? or File -> New -> Project in Visual Studio.
You only told them they need VS installed until this point